inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -100,28 +92,27 @@ def _create_ticket(obj, eng):
         context = {}
         if context_factory:
             context = context_factory(user, obj)
-        body = render_template(
-            template,
-            **context
-        ).strip()
 
         if not in_production_mode():
             obj.log.info(
-                u'Was going to create ticket: {subject}\n\n{body}\n\n'
+                u'Was going to create ticket: {subject}\n'
                 u'To: {requestors} Queue: {queue}'.format(
                     queue=queue,
                     subject=context.get('subject'),
                     requestors=user.email,
-                    body=body
                 )
             )
             return
 
+        recid = obj.extra_data.get("recid") or obj.data.get("control_number") \
+            or obj.data.get("recid")
```

Comment:
You can remove `obj.data.get("recid")`, it's never there (`obj.data` complies with the schema).